src/index.spec.js

```diff
@@ -73,6 +73,22 @@ describe('desciple-core-api', () => {
       expect(verifiedAttestor).to.equal(attestorSsid)
     })
 
+    it('should be able to not verify an attestation of a revoked claim', async () => {
+      let ssid = await discipl.newSsid('memory')
+      await discipl.claim(ssid, { 'need': 'beer' })
+      let claimlink2 = await discipl.claim(ssid, { 'need': 'wine' })
+
+      let attestorSsid = await discipl.newSsid('memory')
+
+      await discipl.attest(attestorSsid, 'agree', claimlink2)
+      await discipl.revoke(ssid, claimlink2)
+
+      let verifiedAttestor = await discipl.verify('agree', claimlink2, [ssid, null, { 'did': 'did:discipl:memory:1234' }, attestorSsid])
```

Comment:
Personally, I'd leave out the invalid/non-attesting ssids and just have attestorSsid, because that's the key one.

Reply:
The verify method will often be called to check for multiple ssids some of which may or may not actually exist in the local situation the method is used in. I want the verify method not to fail on this. But I agree as this is already checked for in other test, will remove the extra ssids.

Reply:
removed the extra ssids